docs/cases/cases-overview.asciidoc

```diff
@@ -18,9 +18,11 @@ external systems from within {es-sec}:
 
 You can create and manage cases via the UI or the <<cases-api-overview>>.
 
-NOTE: To send cases to external systems, you need the
+IMPORTANT: To send cases to external systems, you need the
 https://www.elastic.co/subscriptions[appropriate license] and your role needs *All* privileges for the *Action and Connectors* feature. For information about the base role privileges you need to access cases, refer to <<case-permissions>>.
 
+NOTE: From {es-sec}, you cannot access cases you create in {observability} or Stack Management.
```

Comment:
This could be read as user-specific (the same user who creates the case in Observability cannot then access it in Security), but I assume it doesn't matter which user creates/accesses, it's about where the case was created.
```suggestion
NOTE: From {es-sec}, you cannot access cases created in {observability} or Stack Management.
```